code/game/objects/items/weapons/kitchen.dm

```diff
@@ -65,6 +65,14 @@
 	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 	sharpness = IS_SHARP_ACCURATE
 
+/obj/item/weapon/kitchen/knife/attack(mob/living/carbon/M, mob/living/carbon/user)
+	if(user.zone_selected == "eyes")

```

Comment:
This means it bypasses shield/armour if you aim for the eyes with a knife, doesn't it?